core/unit_test/TestCreateMirror.cpp

```diff
@@ -89,16 +89,16 @@ void test_create_mirror_properties(const View& view) {
   check_memory_space(create_mirror_view(view_alloc(                     DeviceMemorySpace{}), view), DeviceMemorySpace{});
 
   // create_mirror view_alloc + execution space
-  check_memory_space(create_mirror(view_alloc(DefaultExecutionSpace{}, WithoutInitializing),                        view), host_mirror_test_space(view));
-  check_memory_space(create_mirror(view_alloc(DefaultHostExecutionSpace{}),                                         view), host_mirror_test_space(view));
-  check_memory_space(create_mirror(view_alloc(DefaultExecutionSpace{},   WithoutInitializing, DeviceMemorySpace{}), view), DeviceMemorySpace{});
-  check_memory_space(create_mirror(view_alloc(DefaultExecutionSpace{},                        DeviceMemorySpace{}), view), DeviceMemorySpace{});
+  check_memory_space(create_mirror(view_alloc(DefaultHostExecutionSpace{}, WithoutInitializing),                      view), host_mirror_test_space(view));
+  check_memory_space(create_mirror(view_alloc(DefaultHostExecutionSpace{}),                                           view), host_mirror_test_space(view));
```

Comment:
Why is it not just `Kokkos::HostSpace`?

Reply:
OpenMP vs Serial would be distinct execution spaces. I'm tempted to say that Serial should be tested explicitly, along with this

Reply:
I am referring to the trailing argument used in the static assertion

Reply:
The `host_mirror_test_space(view)`

Reply:
If the memory space of the original view is already accessible from the host (HostSpace, CudaUVM, ...), a View in the same memory space is created.

Reply:
for `create_mirror`?  (not `create_mirror_view`)

Reply:
That's what the test already checks, right? Also, see https://github.com/kokkos/kokkos/blob/cc6504d21a0ebc3f17c30aeeaa44c3690fb40d47/core/src/Kokkos_HostSpace.hpp#L193-L219,  https://github.com/kokkos/kokkos/blob/cc6504d21a0ebc3f17c30aeeaa44c3690fb40d47/core/src/Kokkos_View.hpp#L287-L290, https://github.com/kokkos/kokkos/blob/cc6504d21a0ebc3f17c30aeeaa44c3690fb40d47/core/src/Kokkos_View.hpp#L351, and https://github.com/kokkos/kokkos/blob/cc6504d21a0ebc3f17c30aeeaa44c3690fb40d47/core/src/Kokkos_View.hpp#L632-L636.